cypress/components/menu/menu.cy.tsx

```diff
@@ -870,13 +474,12 @@ context("Testing Menu component", () => {
       ["number", 810, 1350, 1350],
       ["string", "700px", "300px", 700],
       ["string", "700px", "1200px", 1200],
-    ])(
+    ] as [string, any, any, number][])(
```

Comment:
why any? 

can we apply `[string, string | number, string | number, number][]`